test/lib/import/colorectal/providers/oxford/oxford_handler_colorectal_test.rb

```diff
@@ -0,0 +1,130 @@
+require 'test_helper'
+
+class OxfordHandlerTest < ActiveSupport::TestCase
+  def setup
+    @record   = build_raw_record('pseudo_id1' => 'bob')
+    @genotype = Import::Brca::Core::GenotypeBrca.new(@record)
+    # TODO: Fully qualify CambridgeHandler in cambridge_handler.rb
```

Comment:
Hi @FrancescoNea, this comment doesn't look related to the Oxford handler?

Reply:
Hi @ollietulloch ,

That's a "typo", apologies.

Oxford Colorectal Coverage test was created from a template for Oxford BRCA coverage test, which in turn was created from Cambridge BRCA coverage test.

Reply:
Thanks @FrancescoNea - no problem, could you push a corrective commit to this branch please?

Reply:
Hi @ollietulloch ,
Resolved by commit [846a3fa](https://github.com/publichealthengland/data_management_system/pull/94/commits/846a3fa7e99e59d877549ccc64a9306c518a83a4)

Reply:
Great, thanks @FrancescoNea